rules/order/src/Rector/Class_/OrderPrivateMethodsByUseRector.php

```diff
@@ -87,18 +76,14 @@ private function call2()
      */
     public function getNodeTypes(): array
     {
-        return [ClassLike::class];
+        return [Class_::class, Trait_::class];
     }
 
     /**
-     * @param ClassLike $node
+     * @param Class_|Trait_ $node
      */
     public function refactor(Node $node): ?Node
     {
-        if ($node instanceof Interface_) {
-            return null;
-        }
-
```

Comment:
@dobryy Tip for simpler code with types. 

In "minus typing" it's sometimes impossible to see what types are expected.

Here it was "all but Interface_..." how many nodes are checked? It's like `!($yes || $maybe)` conditions.

Here it's better to explicitly add types `Class_|Trait_` without "minus typing". This way IDE and PHPStan and human can read it better.